interface/main/finder/dynamic_finder_ajax.php

```diff
@@ -93,30 +165,39 @@
 
 // Column-specific filtering.
 //
+$newWhere = Array();
```

Comment:
Why using this $newWhere array mechanism?

Reply:
For an easier way to build a query string. )